### ISSUE TITLE
Can you publish this on f-droid?

### ISSUE THREAD

**Issue by abdelkadouss** [shiahonb777/web-to-app#308]
This it's an awesome project but i think publish it on f-droid is really the missing part, this gonna insha'Allah increase the accessibility and reliability of the app.

**Comment by moffat-jpg** [comment]
Agreed!!

**Comment by shiaho777** [comment]
I have been paying attention to this issue a long time ago, but because I had no understanding of this platform at that time, I was not ready to put it on the shelves. Now I will prepare it to be put on the shelves at the same time on F-Droid and Google Play

**Comment by shiaho777** [comment]
<img width="3456" height="1924" alt="Image" src="https://github.com/user-attachments/assets/e2025760-9f59-47d9-87b4-fc599cb97ff5" />

**Comment by shiaho777** [comment]
Wait

**Comment by ZMJGMADHPXWT** [comment]
Any news?

**Comment by shiaho777** [comment]
@ZMJGMADHPXWT I will continue to update. During this period, I am waiting for a version with stable feedback, and then start the necessary test version process before it is put on the shelf. That is what Google requires developers. It will take about 14 days, and then it will be officially released. I expect it to be put on Google Play in early October, provided that it passes the review.

**Comment by ZMJGMADHPXWT** [comment]
Is this easier than to put it to F-droid?

**Comment by shiaho777** [comment]
Im not sure. I should list it on f-droid in advance

**Comment by shiaho777** [comment]
Version 2.6.5 is expected to be released in full. Starting from today Beijing time, it will be advanced at a rate of one version per day

**Comment by shiaho777** [comment]
Or multiple versions a day

**Comment by abdelkadouss** [comment]
> Im not sure. I should list it on f-droid in advance

@shiaho777

I believe publishing in f-droid in fact is super easy -even though i never did it my self-, it's just some metadata and then ur ready to push, read the "Submit & Update Apps" section in this article: https://f-droid.org/en/contribute/

**Comment by shiaho777** [comment]
@abdelkadouss Thanks for the information. Ill try it
